include/aa_pipeline_api.hpp

```diff
@@ -338,6 +338,22 @@ namespace astroaccelerate {
 			}
 		}
 
+		size_t range(){
+			return m_ddtr_strategy.range();
+		}
+
+		const int* ndms_data(){
```

Comment:
This should be somethink like get_ndms_array

Reply:
Renamed it as suggested